lib/internal/cluster/master.js

```diff
@@ -104,8 +104,18 @@ function createWorkerProcess(id, env) {
   workerEnv.NODE_UNIQUE_ID = '' + id;
 
   if (execArgv.some((arg) => arg.match(debugArgRegex))) {
-    execArgv.push(`--inspect-port=${process.debugPort + debugPortOffset}`);
-    debugPortOffset++;
+    let inspectPort;
+    if (cluster.settings.inspectPort) {
+      if (typeof cluster.settings.inspectPort === 'function')
```

Comment:
Please add a condition (and test) that throws if `inspectPort` is not a function or number.